src/pages/MainPage.css

```diff
@@ -0,0 +1,37 @@
+header {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0.5rem;
+  width: 95vw;
+  min-width: 280px;
+  color: white;
+}
+
+.Header__topImage {
+  height: 65px;
+
+  background: linear-gradient(300.63deg, #412027 45.56%, #562736 85.31%);
+  border: 2px solid #000000;
+  box-sizing: border-box;
+  border-radius: 30px;
+}
+
+.Header__NavBar {
+  align-self: flex-end;
+  margin-right: 2rem;
+}
+
+.Header__NavBar > * {
+  margin: 0 5px;
+}
+
+footer {
+  height: 100%;
+}
+
+.Footer__Btn {
```

Comment:
Footer_Btn, maybe you could make this className a bit more semantic? 
`Footer__StartButton` or something like that?